homeassistant/components/alexa/capabilities.py

```diff
@@ -1803,6 +1803,13 @@ class AlexaEqualizerController(AlexaCapability):
     """
 
     supported_locales = {"en-US"}
+    valid_sound_modes = {
```

Comment:
We should use all caps to make it clear that this class attribute is a constant.